gm4_dispenser_fireworks/data/dispenser_fireworks/functions/direction/global.mcfunction

```diff
@@ -0,0 +1,12 @@
+#global commands for all directions
+#@s - @e[type=minecraft:firework_rocket,tag=!gm4_df_modified] with dispenser checks succeeded
+#called by dispenser_fireworks:direction/<direction>
+
+#set angle tag
+data merge entity @s {ShotAtAngle:1b}
+
+#add to score
+scoreboard players add @s gm4_df_mod_count 1
```

Comment:
Remove the `global` function and put the nbt in the same merge that applies motion. Copy the scoreboard command in each direction function.

Reply:
I made a `global` function for if something needs to be changed for all directions. It just makes it easier to modify.

Reply:
I get that it makes it easier, but it's a useless function call. If you put the `{ShotAtAngle:1b}` merge inside each direction and you replace the `function dispenser_fireworks:direction/global` function call with `scoreboard players add @s gm4_df_mod_count 1` it functions exactly the same.

Reply:
Argh, alright then